src/Bicep.RegistryModuleTool/Extensions/IEnumerableExtensions.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Bicep.RegistryModuleTool.Proxies;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Bicep.RegistryModuleTool.Extensions
+{
+    internal static class IEnumerableExtensions
```

Comment:
>IEnumerableExtensions [](http://example.com/codeflow?start=26&length=21)

I get what you were going for, but it is unusual for a class to have an "I" prefix. I think in Bicep we called it `EnumerableExtensions` to avoid it.

Reply:
Makes sense. Removed the `I` prefix for all extension class. Btw, in Bicep we also have `IPositionableExtensions` and `IWorkspaceExtensions` which we may need to update as well.

Reply:
Created a good first issue to track it https://github.com/Azure/bicep/issues/5393.